LOHHLAscript.R

```diff
@@ -1647,7 +1659,8 @@ for (region in regions)
       }
       if(!exists('regionSpecOutPut')){
         write.table(paste('\ncoverageStep did not run to completion for: ', HLA_gene, ' in ', region, '! ', '\n', sep = ''), file = log.name, quote = FALSE, row.names = FALSE, col.names = FALSE, append = TRUE)
-        stop('coverageStep did not run to completion!')
```

Comment:
In principle, we could write a file here in the future; I would like better intuition behind `Figures/` though.